Decent.Minecraft.Client/Blocks/Coal.cs

```diff
@@ -11,6 +14,9 @@ protected Coal(bool charred) : base(BlockType.Coal)
         protected bool IsCharred { get; }
     }
 
+    /// <summary>
+    /// <a href="http://minecraft.gamepedia.com/Coal_Ore">Gamepedia link</a>.
+    /// </summary>
     public class Charcoal : Coal
```

Comment:
Ok, will do it, but I still don't get it :( There is no charcoal block in the game and the wiki says:

> Blocks of coal can only be crafted with regular coal, not charcoal.